$NetBSD: patch-preconf_X11ccSunOS_conf,v 1.1 2013/04/14 00:15:27 dholland Exp $

Fix Sun CC configuration to assume at least C89. Otherwise it tries to
use <varargs.h> and fails.

--- preconf/X11ccSunOS/conf~	1995-02-08 19:48:39.000000000 +0000
+++ preconf/X11ccSunOS/conf
@@ -2,7 +2,7 @@ ADDINCLUDEPATHNAME|-I/RW/esprit/X11R5/R6
 ADDLIBPATHNAME|-L/RW/esprit/X11R5/R6/lib/
 ADDLIBSNAME|-lXext -lX11 -lm
 ALIGNMENTNAME|8
-ANSINAME|#undef ANSI_C
+ANSINAME|#define ANSI_C
 BIGLTEXNAME|big-latex
 BINPATHNAME|/usr/local/bin/
 BISONNAME|not_available
